source/administration/important-upgrade-notes.rst

```diff
@@ -2,11 +2,33 @@ Important Upgrade Notes
+|                                                    | A Mattermost user setting to configure desktop notification duration in **Account Settings** > **Notifications** > **Desktop Notifications** is removed.        |
+|                                                    +-----------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | Slash commands configured to receive a GET request will have the payload being encoded in the query string instead of receiving it in the body of the request,  |
+|                                                    | consistent with standard HTTP requests. Although unlikely, this could break custom slash commands that use GET requests incorrectly.                            |
+|                                                    +-----------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | A new ``config.json`` setting to whitelist types of protocols for auto-linking will be added.                                                                   |
+|                                                    | If you rely on custom protocols auto-linking in Mattermost, whitelist them in ``config.json`` before upgrading.                                                 |
+|                                                    +-----------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | A new ``config.json`` setting to disable the `permanent APIv4 delete team parameter                                                                             |
+|                                                    | <https://api.mattermost.com/#tag/teams%2Fpaths%2F~1teams~1%7Bteam_id%7D%2Fput>`_ is added. The setting will be off by default for all new and existing          |
+|                                                    | installs, except those deployed on GitLab Omnibus. If you reply on the APIv4 parameter, enable the setting in `config.json` before upgrading.                   |
+|                                                    +-----------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | An unused ``ExtraUpdateAt`` field will be removed from the channel model.                                                                                       |
```

Comment:
Does this need to be in the upgrade notes if it's an unused field? I may be missing some context here.

Reply:
It wasn't used by the software but someone might have used it via an API